src/test/java/step2/LottosTest.java

```diff
@@ -0,0 +1,18 @@
+package step2;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class LottosTest {
+    @Test
+    @DisplayName("로또 갯수 테스트")
+    void lottosCreateTest() {
+        LottoGenerator lottoGenerator = new LottoGenerator();
+        int money = 6500;
+        Lottos lottos = new Lottos(money, lottoGenerator);
+
+        assertThat(lottos.getLottoList().size()).isEqualTo(money/1000);
```

Comment:
다음과 같이 바로 size()를 테스트할 수 있습니다 😄 
```suggestion
        assertThat(lottos.getLottoList()).hasSize(money / 1000);
```